tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Manage an Endpoint
+summary: Learn how to create, develop, test, publish, and delete an endpoint in a Data App in the TiDB Cloud console.
+---
+
+# Manage an Endpoint
+
+An endpoint in Data Service (beta) is a web API that you can customize to execute SQL statements. You can specify parameters for the SQL statements, such as the value used in the `WHERE` clause. When a client calls an endpoint and provides values for the parameters in a request URL, the endpoint executes the SQL statement with the provided parameters and returns the results as part of the HTTP response.
+
+This document describes how to manage your endpoints in a Data App in the TiDB Cloud console.
+
+## Before you begin
+
+Before you manage an endpoint, make sure that you have created a cluster and a Data App. For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md) and [Create a Data App](/tidb-cloud/data-service-manage-data-app.md#create-a-data-app).
+
+Before you call an endpoint, make sure that you have created an API key in the Data App. For more information, see [Create an API key](/tidb-cloud/data-service-manage-data-app.md#create-an-api-key).
+
+## Create an endpoint
+
+To create an endpoint, perform the following steps:
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/dataservice) page of your project.
+2. In the left pane, click the name of your target Data App and click **...** > **Create Endpoint**.
+
+    <!--TODO: Add the icon to MDSvgIcon.tsx-->
+
+    > **Tip:**
+    >
+    > Alternatively, you can also locate the Data App and click <svg width="16" height="16" viewBox="0 -2 24 24" fill="none" stroke-width="3" xmlns="http://www.w3.org/2000/svg"><path d="M12 5V19M5 12H19" stroke="currentColor" stroke-width="inherit" stroke-linecap="round" stroke-linejoin="round"></path></svg> **Create Endpoint** at the top of the left pane.
+
+3. The newly created endpoint is added to the top of the endpoint list.
+
+## Develop an endpoint
+
+For each endpoint, you can write SQL statements to execute on a TiDB cluster, define parameters for the SQL statements, or manage their name and version.
+
+### Configure properties
+
+On the right pane of the endpoint details page, you can click the **Properties** tab to view and manage the following properties of the endpoint:
+
+- **Path**: the unique path of the endpoint that users use to access it. The path should be unique within a Data App and is appended to the service URL of the endpoint.
+
+    - To obtain the service URL of the endpoint, hover your mouse over **SERVICE_URL**. For example, `https://data.tidbcloud.com/api/v1beta/apps/{APP_ID}/endpoint`.
+    - Only letters, numbers, and underscores (`_`) are allowed in the path, which must start with a slash (`/`). For example, `/my_endpoint`.
```

Comment:
The path must be a maximum of 64 characters in length and can contain letters, digits, underscores (_), forward slashes (/) and must start with a forward slash (/)